app-tracking-protection/vpn-api/src/main/java/com/duckduckgo/mobile/android/vpn/VpnFeaturesRegistry.kt

```diff
@@ -63,6 +63,8 @@ interface VpnFeaturesRegistry {
     fun registryChanges(): Flow<Pair<String, Boolean>>
 
     fun getRegisteredFeatures(): List<VpnFeature>
+
+    fun getLastRegistryChange(): Pair<String, Boolean>
```

Comment:
The implementations of `VpnStaticNotificationContentPlugin` wouldn't have a good place to subscribe to the  Flow returned by `registryChanges` (returns hot flow). We need to be able to get whatever is the last known registryChange  when isActive is called.

Reply:
Can we do `registryChanges().last()` instead? I still don't see why we have to introduce the `getLastRegistryChange()` method having the `Flow` already